src/drivers/boards/px4fmu-v4pro/board_config.h

```diff
@@ -297,8 +297,7 @@ __BEGIN_DECLS
 #define GPIO_PWM_IN			GPIO_TIM4_CH2IN_2
 
 #define GPIO_RSSI_IN 			(GPIO_INPUT|GPIO_PULLUP|GPIO_PORTC|GPIO_PIN1)
-#define GPIO_LED_SAFETY			(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTC|GPIO_PIN3)
-#define GPIO_BTN_SAFETY			(GPIO_INPUT|GPIO_PULLUP|GPIO_PORTC|GPIO_PIN4)
```

Comment:
There are two issues, this resolves the first: The first is that an unconnected signal had the same name as the safety switch input. By changing its name here it disables the fmu from treating this input as the safety switch.